pkg/logcli/query/query.go

```diff
@@ -196,7 +200,23 @@ func (q *Query) DoLocalQuery(out output.LogOutput, statistics bool, orgID string
 	conf.StorageConfig.BoltDBShipperConfig.Mode = indexshipper.ModeReadOnly
 	conf.StorageConfig.BoltDBShipperConfig.IndexGatewayClientConfig.Disabled = true
 
-	querier, err := storage.NewStore(conf.StorageConfig, conf.ChunkStoreConfig, conf.SchemaConfig, limits, cm, prometheus.DefaultRegisterer, util_log.Logger)
+	schema := conf.SchemaConfig
+	if useRemoteSchema {
+		cm := storage.NewClientMetrics()
+		client, err := GetObjectClient(conf, cm)
+		if err != nil {
+			return err
+		}
+
+		config, err := LoadFromURL(client)
```

Comment:
`config` is mislead. How about 
```suggestion
		loadedSchema, err := LoadFromURL(client)
```

Reply:
Yes, that's better. This was applied manually.